java/client/src/main/java/glide/api/models/commands/LPosOptions.java

```diff
@@ -0,0 +1,53 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+/**
+ * Optional arguments to {@link glide.api.commands.ListBaseCommands#lpos(java.lang.String,
+ * java.lang.String, glide.api.models.commands.LPosOptions)} and {@link
+ * glide.api.commands.ListBaseCommands#lposCount(java.lang.String, java.lang.String, long,
+ * glide.api.models.commands.LPosOptions)} command.
+ *
+ * @see <a href="https://redis.io/commands/lpos/">redis.io</a>
+ */
+@Builder
```

Comment:
I propose to use a constructor (or few) for such simple structure - it will simplify the user API.

Reply:
The only problem is that she won't be able to separate rank and maxLength (both longs) so she would have a hard time creating a constructor for ONLY rank and ONLY maxLength

Reply:
So it is possible to  do with factory methods.